pkg/rpc/server/server.go

```diff
@@ -36,7 +44,16 @@ func ListenAndServe(addr string, c config.Config) error {
 		})
 	}
 
-	go updateDBWorker(c.AppVersion, c.CacheDir, dbUpdateWg, requestWg)
+	go func() {
+		worker := initializeDBWorker()
```

Comment:
Just for my own understanding: The only advantage of using `google/wire` here is that we can inline calls like these rather than initializing a bunch of dependencies before using them. Is that correct or is there more to it?

Reply:
Exactly. `google/wire` is a small tool. And, we can define dependencies in advance by `wire.NewSet`. It improves the readability.